tensorflow_addons/layers/noisy_dense.py

```diff
@@ -33,16 +33,22 @@ def _scale_noise(x):
 
 @tf.keras.utils.register_keras_serializable(package="Addons")
 class NoisyDense(tf.keras.layers.Dense):
-    r"""Noisy dense layer that inject random noise to
-    the weights of normal dense layer.
+    r"""Noisy dense layer that injects random noise to the weights of dense layer.
+
+    Noisy dense layers are fully connected layers whose weights and biases are
+    augmented by factorised Gaussian noise. The factorised Gaussian noise is
+    controlled through gradient descent by a second weights layer.
 
     A `NoisyDense` layer implements the operation:
     $$
     \mathrm{NoisyDense}(x) =
-    \mathrm{activation}(\mathrm{dot}(x, \mu + (\sigma \cdot \eps))
+    \mathrm{activation}(\mathrm{dot}(x, \mu + (\sigma \cdot \epsilon))
     + \mathrm{bias})
     $$
-    with bias only being added if `use_bias` is `True`.
+    Where $\mu$ is the standard weights layer, $\epsilon$ is the factorised
```

Comment:
NIT: lower case `w` -> `where`

Reply:
Oops, I forgot to change this in the committed version. All fixed!